core/lib/config/src/configs/contracts.rs

```diff
@@ -14,6 +14,13 @@ pub enum ProverAtGenesis {
 pub struct ContractsConfig {
     pub governance_addr: Address,
     pub mailbox_facet_addr: Address,
+
+    pub bridgehub_proxy_addr: Address,
```

Comment:
please add comments (maybe also with names of the interfaces that should be deployed there)